docker/orb.yaml

```diff
@@ -427,12 +427,12 @@ jobs:
           builds faster by skipping steps that have not changed.
         type: boolean
       executor:
-        default: cimg
+        default: cimg/base:current
```

Comment:
This references the executor _name_ not _image_ (ie. line 9, not 11). Keep it as `cimg`.

Reply:
oops! Change it accordingly